protocols/gossipsub/src/types.rs

```diff
@@ -474,7 +474,7 @@ impl From<RpcOut> for proto::RPC {
 
 /// An RPC received/sent.
 #[derive(Clone, PartialEq, Eq, Hash)]
-pub struct Rpc {
+pub struct RpcIn {
```

Comment:
What's the purpose of renaming it?

Reply:
Thanks for asking Elena, the goal is to be able to now make explicit the difference with `RpcOut`:
https://github.com/libp2p/rust-libp2p/blob/054c825b49dc782b035c231f26530b885a9bef99/protocols/gossipsub/src/types.rs#L317-L341